app/grants/models.py

```diff
@@ -1994,3 +1994,19 @@ class GrantStat(SuperModel):
 
     def __str__(self):
         return f'{self.snapshot_type} {self.created_on} for {self.grant.title}'
+
+
+class GrantBrandingRoutingPolicy(SuperModel):
```

Comment:
@aamustapha you should describe the model (docstring) here

Reply:
@chibie good thinking - @aamustapha its not super important but since this model is not super clear as to what it does it would be helpful. You can check the `PhantomFunding` model in this file for an example.